downloadVid.py

```diff
@@ -0,0 +1,34 @@
+import os
+from pytube import YouTube
+import requests
+from tqdm import tqdm
+import math
+
+def download_video(video_url):
+    yt = YouTube(video_url)
+    stream = yt.streams.filter(file_extension='mp4').first()
+    response = requests.get(stream.url, stream=True)
+    filename = yt.title + ".mp4"
+    file_path = os.path.join("Video/", filename)
+
+    #If file already exists, don't download it again
+    if os.path.isfile(file_path):
```

Comment:
Added the checker for already downloaded videos.
It's pretty slow tbh and I don't know why but it works.